doc/samples/samples_list.rst

```diff
@@ -11,22 +11,20 @@ The following table demonstrates the variants' configuration for the supported d
 |                                   |                                          | * `Thingy53`_                  |                                                                           |                                               |                           |
-|                                   |                                          | * `nRF54L15 PDK`_ `*`          |                                                                           |                                               |                           |
+|                                   |                                          | * `nRF54L15 DK`_               |                                                                           |                                               |                           |
 +-----------------------------------+------------------------------------------+--------------------------------+---------------------------------------------------------------------------+-----------------------------------------------+---------------------------+
-| :ref:`variant_sidewalk_dut`       | Device under test generic application    | * `nRF52840 DK`_               | * External flash (included in the DK) except `nRF54L15 PDK`_              | * MCUboot - DFU partition in external flash   | * ``overlay-dut.conf``    |
+| :ref:`variant_sidewalk_dut`       | Device under test generic application    | * `nRF52840 DK`_               | * External flash (included in the DK) except `nRF54L15 DK`_               | * MCUboot - DFU partition in external flash   | * ``overlay-dut.conf``    |
 |                                   | having activated CLI.                    | * `nRF5340 DK`_                | * `Semtech SX1262MB2CAS`_ (only for the configurations with LoRa or FSK)  | * Application partition size = 956 kB         |                           |
-|                                   |                                          | * `nRF54L15 PDK`_ `*`          |                                                                           |                                               |                           |
+|                                   |                                          | * `nRF54L15 DK`_               |                                                                           |                                               |                           |
 +-----------------------------------+------------------------------------------+--------------------------------+---------------------------------------------------------------------------+-----------------------------------------------+---------------------------+
 
-`*` The `nRF54L15 PDK`_ does not support LoRa and FSK configurations.

```

Comment:
@ktaborowski removed the note, let me know if this is still applicable

Reply:
nrf54l15 dk supports all transports now.

Reply:
leaving as is then, thank you!